.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
add the entire .vs folder to .gitignore and remove these files and folders from the commit